opm/input/eclipse/Schedule/UDQ/UDQASTNode.hpp

```diff
@@ -74,6 +74,13 @@ class UDQASTNode
     bool operator==(const UDQASTNode& data) const;
     void required_summary(std::unordered_set<std::string>& summary_keys) const;
 
+    /// Publish collection of requisite objects needed to evaluate this node.
```

Comment:
One more here

Reply:
> One more here

Right.  I've pushed an update to address this.